src/index.ts

```diff
@@ -90,30 +103,31 @@ export class Executer {
           this.clientObjs
             .get(key)
             .call(actionObj.information)
-            .then(getSetResponse => {
+            .then(response => {
               // Action succeeded
               // Commented out due to excesively large response with getSchema
-              // pamLog('Received response from the radio: %s', getSetResponse)
+              // pamLog('Received response from the radio: %s', response)
               actionObj.information.response = {
-                data: getSetResponse,
+                data: response,
                 error: null
               }
               resolve(actionObj.serialize())
             })
-            .catch(getSetError => {
-              // Axios error (connection refused), invalid action type, Not logged in
-              pamLog('Received the following ERROR: %s', getSetError)
-              /* istanbul ignore next */
+            .catch(error => {
+              pamLog('Received the following ERROR: %s', error)
               actionObj.information.response = {
-                error: getSetError.toString(),
+                error: error,
                 data: null
               }
               reject(actionObj.serialize())
```

Comment:
I was under the impression that we would be 'resolving' with these error action objects. If we still plan on rejecting with the action objects I'm going to need to adjust how I'm doing the CSM error handling, and the frontend is going to need some adjustments as well.

Reply:
For simplicities sake, I'm going to continue right now with the assumption that we will still be rejecting errors. I don't think the effort in the frontend will be all that difficult, and it will actually make my life easier in CSM.

Reply:
Ok sounds good